requirements.in

```diff
@@ -41,3 +41,4 @@ requests-cache
 requests>=2.20.0
 sentry-sdk
 six
+uwsgi
```

Comment:
Linkedevents itself does not need uwsgi. I think it would be better installed in the Dockerfile. In fact, I think it is a better idea to source uwsgi from Ubuntu repositories. They seem to be doing all sorts of tweaks and polishing: http://changelogs.ubuntu.com/changelogs/pool/universe/u/uwsgi/uwsgi_2.0.18-11ubuntu1/changelog

Reply:
I see, that's right. Reverted the installation in requirements file (95a858b113549a0a884e52025438bbd5f80ad59c) , and added uwsgi to Dockerfile (9cb86c4fb673fe900c9b1f6e690724c32884fb62) 👍🏼